Makefile

```diff
@@ -707,7 +707,8 @@ RUNTIME_CPP_COMPONENTS = \
   windows_threads_tsan \
   windows_yield \
   write_debug_image \
-  x86_cpu_features
+  x86_cpu_features \
+  hexagon_cache_allocator
```

Comment:
This also needs to be added to the CMake build

Reply:
Done